apps/browser/src/background/main.background.ts

```diff
@@ -568,7 +583,9 @@ export default class MainBackground {
     this.twoFactorService.init();
 
     await this.tabsBackground.init();
-    await this.contextMenusBackground.init();
+    if (!this.popupOnlyContext) {
+      this.contextMenusBackground.init();
```

Comment:
Seems smart to handle the null here, since you're not always instantiating the class.

```suggestion
      this.contextMenusBackground?.init();
```